tests/page_objects/helioviewer.ts

```diff
@@ -312,6 +328,89 @@ class Helioviewer {
         await this.page.locator('#logo').hover();
     }
 
+    /**
+    * Get the loaded date in helioviewer 
+    * @returns Date|null - Loaded date of helioviewer, it can be null if any error.
+    */
+    async GetLoadedDate(): Date|null {
+
+        const currentDate = await this.page.getByLabel('Observation date', {exact:true}).inputValue();
+        const currentTime = await this.page.getByLabel('Observation time', {exact:true}).inputValue();
+
+        const date = new Date(currentDate +' '+ currentTime);
+
+        if (isNaN(date.getTime())) {
+            return null;
+        }
```

Comment:
This is bad, right? Should we just move the expect here?

```suggestion
await expect(date.getTime()).not.toBeNull();
```